### PR TITLE
Add switch to control `debug=true` on WCOSS2 for development testing

### LINKED ISSUE #619: Add runtime debug option to setup scripts
**Description**

This issue adds the ability to add the debug flag to scheduler submissions when needed. The following flag, when used on WCOSS2, runs jobs in debug mode and provides additional information in the logs:

```
#PBS -l debug=true
```

Developers would like to be able to easily toggle on/off this scheduler flag when testing. It can be added into the rocoto xml or, in the future, the ecflow script scheduler directives.

A new debug option should be added to `config.base` to toggle on/off the option and which will then add the needed debug flag for the respective scheduler into the resulting rocoto xml.

**Requirements**

This is a new feature.

**Acceptance Criteria (Definition of Done)**

A user/developer can provide a debug flag to the setup script command and get the needed debug setting in the resulting generated rocoto xml.

**Dependencies**

No dependencies.

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
# Description

This PR adds a switch (`DEBUG_POSTSCRIPT`) to control whether `debug=true` is set when submitting development rocoto jobs to PBS schedulers (currently just WCOSS2). This switch added to `config.base` will only work and be invoked on PBS platforms (e.g. WCOSS2). There isn't an equivalent flag to set for SLURM on the RDHPCS. Have added this new switch to documentation.

The `DEBUG_POSTSCRIPT` switch in `config.base` will be set to off/`"NO"` by default.

This PR does not touch the `debug=true` settings in the ecf scripts, which should remain for ops per NCO requests.

Resolves #619

# Type of change

- New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?

Low res tests on Hera and WCOSS2.

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: parm/config/gefs/config.base [hunk 1]
```diff
@@ -75,6 +75,7 @@ export NMV="/bin/mv"
 export NLN="/bin/ln -sf"
 export VERBOSE="YES"
 export KEEPDATA="NO"
+export DEBUG="NO" # Only works on WCOSS2
```
Comment (aerorahul) [inline parm/config/gefs/config.base]: `DEBUG_SCHEDULER` is perhaps better suited than a generic `DEBUG`. What do you think?

**Review by aerorahul — APPROVED** [review]
Looks good.
I would suggest using a targeted `DEBUG_xyz` to make it clear, this is debugging `xyz` and not a generic debug flag.

File: parm/config/gefs/config.base (hunk 1 above)
Comment (KateFriedman-NOAA) [reply parm/config/gefs/config.base]: Have changed to `DEBUG_SCHEDULER` per reviewer suggestion and offline discussion.

**Comment by HenryRWinterbottom** [comment]
@KateFriedman-NOAA This looks good to me. 

Just a FYI, the following exists for SLURM: [OPT_DebugFlags](https://slurm.schedmd.com/slurm.conf.html#OPT_DebugFlags).

It is more convoluted than the above, but this is more intended as a FYI and beyond the scope of this PR (and likely anything else we might need).

**Comment by KateFriedman-NOAA** [comment]
Thanks @HenryWinterbottom-NOAA ! I spoke with the Hera sysadmins about those options too but they are also meant for the admin side of the scheduler and not for users.

**Comment by HenryRWinterbottom** [comment]
> Thanks @HenryWinterbottom-NOAA ! I spoke with the Hera sysadmins about those options too but they are also meant for the admin side of the scheduler and not for users.

@KateFriedman-NOAA That makes sense. The nerd in me has SLURM running on a small cluster here, so I forgot that it required admin permissions. Thank you for the heads up.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**